mmpose/core/evaluation/top_down_eval.py

```diff
@@ -250,6 +250,83 @@ def _taylor(heatmap, coord):
     return coord
 
 
+def post_dark(coords, batch_heatmaps, share_heatmap=True):
+    """DARK post-pocessing.
+
+    Note:
+        batch_size: N
+        num_keypoints: K
+
+    Args:
+        coords (np.ndarray[N, K, 2]): Coords in shape of batchsize*num_kps*2.
+        batch_heatmaps (np.ndarray[N, K, H, W]): batch_heatmaps
+        share_heatmap (bool) : True for bottom_up paradigm where all persons
+                    share the same heatmap while False for top_down paradigm
+                    where each person has its own heatmaps.
+
+    Returns:
+        res (np.ndarray): Refined coords in shape of batchsize*num_kps*2.
+    """
+    if not isinstance(batch_heatmaps, np.ndarray):
+        batch_heatmaps = batch_heatmaps.cpu().numpy()
+    batch, num_kps, h, w = batch_heatmaps.shape
+    persons = coords.shape[0]
```

Comment:
use `B = coords.shape[0]`

Both B and N can be used to represent batch size.

Reply:
'Batch' and 'persons' are different in bottom-up paradigm

Reply:
Then need to update the docstring?

![image](https://user-images.githubusercontent.com/9464825/100878798-9c567900-34e5-11eb-8db0-f53de37d146b.png)

Reply:
yes